test/test_mimic_explainer.py

```diff
@@ -392,6 +394,29 @@ def test_explain_model_imbalanced_classes(self, mimic_explainer):
         assert np.isclose(surrogate_predictions, model_predictions).all()
         assert global_explanation.method == LIGHTGBM_METHOD
 
+    def test_explain_raw_feats_regression(self, mimic_explainer):
+        # verify that no errors get thrown when calling get_raw_feat_importances
+        num_features = 19999
+        num_rows = 1000
+        test_size = 0.2
+        X, y = make_regression(n_samples=num_rows, n_features=num_features)
+        x_train, x_test, y_train, y_validation = train_test_split(X, y, test_size=test_size, random_state=42)
```

Comment:
Should probably keep the naming consistent:
- capital X and lowercase y
- either test or valid, but not both

(I recommend X_train, X_valid, y_train, y_valid)

Reply:
good point, I changed y_validation to _ because I don't use it. For capital/lowercase, actually this is the only file that uses capital X due to external contributions.  I think we decided a while ago to use lowercase x_train/x_test, even though I'm aware in automl codebase this is different.  However, I'm open to changing it to be X_train in a separate PR.